File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/createnew/dialogs/askforname/AskForNameListener.java
package ua.meugen.android.pdfcomposer.ui.activities.createnew.dialogs.askforname;

import java.util.List;

import ua.meugen.android.pdfcomposer.model.data.PageContent;

public interface AskForNameListener {

    void onDocumentNamed(String name, List<PageContent> pages);
}

File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/createnew/dialogs/askforname/AskForNameListener.kt
package ua.meugen.android.pdfcomposer.ui.activities.createnew.dialogs.askforname

import ua.meugen.android.pdfcomposer.model.data.PageContent

interface AskForNameListener {

    fun onDocumentNamed(name: String, pages: List<PageContent>)
}
